spring-data/function-invocation/README.md

```diff
@@ -0,0 +1,44 @@
+# Function Invocation Example
+
+## Description
+In this example a GemFire client will invoke remote functions registered on a GemFire server.
+
+## Prerequisites
+In order to run the following example one requires the following to be set up:
+* Java 17
+* Latest VMware GemFire is installed locally
+* Set up local `gradle` or `maven` environment by following Steps 1-6 and 8 from [Spring Boot For VMware GemFire Quickstart](https://docs.vmware.com/en/Spring-Boot-for-VMware-GemFire/index.html#spring-boot-for-vmware-gemfire-quick-start-0)
+
+## Steps to run
+1. Clone `Spring For GemFire Examples` project
+2. Start GemFire server-side processes
+    1. Open new Terminal
+    2. `cd` into the `function-invocation` project directory, `spring-examples/spring-data/function-invocation`
+    3. Start `gfsh` located in the `{VMware_GemFire_installation_location}/bin` directory
+    4. Launch the Locator and Server by running `start.gfsh` file located in the `function-invocation` project directory in `gfsh`. <br> e.g `run --file={pathToStartGfshFile}/start.gfsh`
+    5. Confirm that the locator and server are running by running `list members` in `gfsh`, which should contain two entries.
+3. Start Client
```

Comment:
We seem to possibly be missing a step to confirm that the functions had been correctly deployed.

Reply:
I can add a `list functions` step